frontend_tests/node_tests/voting_widget.js

```diff
@@ -76,4 +83,202 @@ run_test('poll_data_holder my question', () => {
         question: 'best plan?',
     });
 
+    const invalid_vote_event = {
+        type: 'vote',
+        key: '98,1',
+        vote: 1,
+    };
+
+    blueslip.error = (msg) => {
+        assert.equal(msg, `unknown key for poll: ${invalid_vote_event.key}`);
+    };
+    data_holder.handle_event(sender_id, invalid_vote_event);
+    data = data_holder.get_widget_data();
+
+    const comment_outbound_event = data_holder.handle.new_comment.outbound('new comment');
+    assert.deepEqual(comment_outbound_event, {
+        type: 'new_comment',
+        idx: 2,
+        comment: 'new comment',
+    });
+
+    const new_question = 'Any new plan?';
+    const question_outbound_event = data_holder.handle.question.outbound(new_question);
```

Comment:
Here I think we should also test if that the `question` event is only being created by the user who started the poll. (re: Reply on first commit)